static/src/js/components/NotAvailableIcon/NotAvailableIcon.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const NotAvailableIcon = ({ size }) => (
+  <svg
+    width={size}
+    height={size}
+    viewBox="0 0 20 20"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+    className="meta-icon__icon"
+    role="graphics-symbol"
```

Comment:
I think we probably still want an `aria-label` on this svg. Something like "A slash icon" is probably fine.